src/mapping/tests/AxialGeoMultiscaleMappingTest.cpp

```diff
@@ -0,0 +1,130 @@
+#include <Eigen/Core>
+#include <algorithm>
+#include <memory>
+#include "logging/LogMacros.hpp"
+#include "mapping/AxialGeoMultiscaleMapping.hpp"
+#include "mapping/Mapping.hpp"
+#include "math/constants.hpp"
+#include "mesh/Data.hpp"
+#include "mesh/Mesh.hpp"
+#include "mesh/SharedPointer.hpp"
+#include "mesh/Utils.hpp"
+#include "mesh/Vertex.hpp"
+#include "testing/TestContext.hpp"
+#include "testing/Testing.hpp"
+
+using namespace precice;
+using namespace precice::mesh;
+
+BOOST_AUTO_TEST_SUITE(MappingTests)
+BOOST_AUTO_TEST_SUITE(AxialGeoMultiscaleMapping)
+
+BOOST_AUTO_TEST_CASE(testConsistentSpread)
+{
+  PRECICE_TEST(1_rank);
+  int dimensions = 3;
+  using testing::equals;
+
+  // Create mesh to map from
+  PtrMesh inMesh(new Mesh("InMesh", dimensions, testing::nextMeshID()));
+  PtrData inData    = inMesh->createData("InData", 3, 0_dataID);
+  int     inDataID  = inData->getID();
+  Vertex &inVertex0 = inMesh->createVertex(Eigen::Vector3d::Constant(0.0));
+  inMesh->allocateDataValues();
+  Eigen::VectorXd &inValues = inData->values();
+  inValues << 0.0, 0.0, 2.0;
+
+  double radius = 1.0;
```

Comment:
We probably need a comment for the intention/role of the `radius`. I can imagine that later on we will have more options, or that this test ends up being debugged by someone who does not know much about this mapping method.

Reply:
Okay, I will try to document everything more thoroughly!